### ISSUE TITLE
Balances are Not Tracked

### ISSUE THREAD

**Issue by bmino** [bmino/binance-triangle-arbitrage#27]
After executing a trade, balance deltas are not being displayed.

This is caused by two variables being compared which are references instead of values.
`initialBalance` and `this.balances` are equivalent through their lifetime

**Comment by vesuvius729** [comment]
This seems correct, I was having this problem earlier today as well. I gave an account 0.03 BTC to play with, and let it trade up to 0.02 BTC at a time. From my log:

```
{"level":"info","time":"2/13/2019, 5:13:38 PM","msg":"Buying 4 NEOBTC @ market price","v":1}
{"level":"info","time":"2/13/2019, 5:13:39 PM","msg":"Selling 4 NEOBNB @ market price","v":1}
{"level":"info","time":"2/13/2019, 5:13:39 PM","msg":"Selling 3.63 BNBBTC @ market price","v":1}
{"level":"info","time":"2/13/2019, 5:13:39 PM","msg":"Executed BTC-NEO-BNB position in 995 ms","v":1}
{"level":"info","time":"2/13/2019, 5:39:17 PM","msg":"Buying 7834 FUELBTC @ market price","v":1}
{"level":"info","time":"2/13/2019, 5:39:18 PM","msg":"Selling 7834 FUELETH @ market price","v":1}
{"level":"info","time":"2/13/2019, 5:39:18 PM","msg":"Selling 0.499 ETHBTC @ market price","v":1}
{"level":"error","time":"2/13/2019, 5:39:18 PM","msg":"Account has insufficient balance for requested action.","v":1}
```

Would love to have a quick-fix for this if you've got it :)

**Comment by bmino** [comment]
Thanks for your interest! Please report anything you see with a new issue and I'll take a look.

This is fixed in v4.4.0 :D
